shared/middleware/helpers/auth_helpers.rb

```diff
@@ -18,6 +18,14 @@ def current_user
   @dashboard_user ||= DASHBOARD_DB[:users][id: current_user_id]
 end
 
+# Returns true if the current user is under 13 or if age is unknown.
+def under_13?
+  return true unless current_user
+  birthday = current_user[:birthday]
+  age = ((Date.today - birthday) / 365).to_i
+  age < 13
+end
```

Comment:
To answer your question, the only line I see a way to easily share is `age = ((Date.today - birthday) / 365).to_i`, and I don't think that's enough complexity to warrant sharing. I'd be happy to add a comment explaining that the logic is duplicated.